bookwyrm/forms.py

```diff
@@ -242,14 +243,39 @@ def clean(self):
         filetype = cleaned_data.get("filetype")
         book = cleaned_data.get("book")
         domain = urlparse(url).netloc
+
+        try:
+            response = requests.head(url, timeout=3)
+
+            if response.status_code in range(300, 308):
+                headers = response.headers
+                url_redirect = headers["Location"]
+                print(url_redirect)
+
+                response = requests.head(url_redirect, timeout=3)
+            if int(response.status_code) < 200 or int(response.status_code) > 299:
+                # status code between 200 and 399 should be fine
```

Comment:
This comment is inconsistent with the condition above it, one of them should be updated (are 3xx codes okay or not?)